intermediate_source/dqn_with_rnn_tutorial.py

```diff
@@ -342,7 +342,10 @@
 # will return a new instance of the LSTM (with shared weights) that will
 # assume that the input data is sequential in nature.
 #
-policy = Seq(feature, lstm.set_recurrent_mode(True), mlp, qval)
+from torchrl.modules import set_recurrent_mode
+
+with set_recurrent_mode(True):
```

Comment:
This is to be set during the loss computation. 

The policy module given to the collector and to the loss will be the same. The decorator is to be set to True whenever a recurrent call is required (typically within the loss).

Reply:
Can you check the update, @vmoens ?